libs/vue/src/builders/dev-server/builder.ts

```diff
@@ -89,8 +102,10 @@ export function runBuilder(
           config.plugin('vue-cli').use({
             apply: (compiler) => {
               compiler.hooks.afterEnvironment.tap('vue-cli', () => {
-                // eslint-disable-next-line @typescript-eslint/no-empty-function
-                compiler.watchFileSystem = { watch: () => {} };
+                compiler.watchFileSystem = {
+                  // eslint-disable-next-line @typescript-eslint/no-empty-function
+                  watch: () => {},
+                };
```

Comment:
Remove these changes.

Reply:
I think it got these changes when the prettier formatting touched it. I will double check this.